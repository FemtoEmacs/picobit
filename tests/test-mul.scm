;; very simple sanity test for bignums
(define (foo x y)
  (* x y))

;(foo 1 3)
(display (if (= (foo 15 3) 45) ; TODO test with bignum-range numbers
	     "OK\n"
	     "NAH\n"))
