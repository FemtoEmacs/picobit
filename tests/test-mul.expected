:0080000080
:10800000FBD72503811C40008122400081184000DD
:10801000810E4000810F400081114000810B400023
:108020008024011B8071011D806E011E8027011FAD
:108030008027012080420002805301218024000219
:10804000806A000280730123806701248077012504
:108050008073012680270127800E000280450128B9
:108060008032011280270113806601148069011596
:108070008040011680400117804F01198052011A7B
:108080008070011080780110807D010C8066010DE8
:1080900080780002804C011902B402115102B4012F
:1080A000925002B4018C521307B00187A031C7B3BC
:1080B0000182A109B582EED80120DAB8954221DB10
:1080C000B6800221CC9A21CE2161EE2022CFB67259
:1080D00000EF20B6800120DA9EA02607E620B558E2
:1080E000EEA02607E600EF20C0B8BF20B582B64854
:1080F000012004C8B8AEA03121B5A402B583CDDCFF
:10810000EF02A034210EC5C122CD2102CDB4011150
:108110007101210EC89220EF210E406221B662014A
:108120000421C2EF2002B559B75520CCB8B8A02C15
:1081300007E620CEB51FEE20CFB6800120D296A054
:108140002D07E600EF20CC9DA02407E620CEB50544
:10815000EE20CFB666A10AB000B8EE20B000D5EE92
:10816000A02D07E600EF20D995A107B100B820F0B7
:108170009C2095A106B100B8A104B100B8A108B136
:1081800000B8A105B200B40220224172012007E626
:1081900000EF022004C8212302CDCDB4019F71015C
:1081A0002204C822242302CDCDCDB401AE71012317
:1081B000B8DC21B0011F23222602CDCDCDB401C1F0
:1081C000710122B8C621B0011F21242602CDCDCDD8
:1081D000B401D471012022242602CDCDCDCDB4012D
:1081E000E270002121C4232502CDCDB401EF71013D
:1081F00022B893219220EF2298019420B1011F20F0
:10820000EF21B87BB77500B86EB76A21B73B21B7CD
:1082100025FE21CCB302AB2022B6800221CCB302D2
:10822000A92022CEB58422CFB671022004C8212312
:1082300002CDCDB4023771012204C822242302CD1D
:10824000CDCDB40246710123B8DC21B0011F232239
:108250002602CDCDCDB40259710122B8C621B0019C
:108260001F21242602CDCDCDB4026C710120222421
:108270002602CDCDCDCDB4027A70002121C32325B5
:1082800002CDCDB40287710122B893219220EF2252
:1082900098019420B1011F20EF21B87BB77500B879
:0D82A0006EB76A21B73B21B72520EF20EF14
:00000001FF
